### ISSUE TITLE
No modification reason on RC.

### ISSUE THREAD

**Issue by Paddy3118** [dkandalov/rosettacode-kotlin#5]
This flow should be improved to give proper modification reasons on Rosetta Code rather that the current "see this repo" which requires further searching to see if anyone deigned to comment for RC.

**Comment by dkandalov** [comment]
I agree that change comments could be better. Are there guidelines specific to RC or any reasonable comments like for version control is enough?

**Comment by dkandalov** [comment]
Reddit thread for reference: https://www.reddit.com/r/Kotlin/comments/6yhqiq/please_be_kind_to_rosettacode/
